source/schemas/schemas.py

```diff
@@ -0,0 +1,14 @@
+from marshmallow import Schema, fields
+
+
+class UserSchema(Schema):
+    id_user = fields.Integer()
+    username = fields.String()
+
+
+class TaskSchema(Schema):
+    id_task = fields.Integer()
+    task_name = fields.String()
+    task_description = fields.String()
+    date_created = fields.DateTime()
+    id_task_status = fields.Integer()
```

Comment:
Maybe you can create a schema for `task status` and instead of show the id show the entire status object so the status string is included in the response. 
In order to do this you have to:
1) add an attribute to the task model that is named task_status but will not be a column but a relationship
2) Add a TaskStatusSchema with the id and the description
3) Change this field for the `task_status` field and use the schema to show the `status` object like: `task_status = fields.Nested(fields.TaskStatus())`